cfme/tests/v2v/test_v2v_manual.py

```diff
@@ -0,0 +1,312 @@
+"""Manual tests"""
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [test_requirements.v2v, pytest.mark.manual]
+
+
+@pytest.mark.tier(2)
+def test_osp_test_flavors_can_be_selected_creating_migration_plan():
+    """
+    title: OSP: Test flavors can be selected creating migration plan
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan amd choose flavors
+            3. Start migration
+        expectedResults:
+            1. Mapping created and visible in UI
+            2. Plan created and visible in UI
+            3. Successful migration from vmware to OSP with the selected flavor
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_creating_multiple_migration_plans_with_same_name():
+    """
+    title: OSP: Test duplicate names in creating migration plan
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: medium
+        caseposneg: negative
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create two migration plan with same name
+        expectedResults:
+            1. Mapping created and visible in UI
+            2. Duplicate migration plan name should not be allowed.
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_if_no_password_is_exposed_in_logs_during_migration():
+    """
+    title: OSP: Test if no password is exposed in logs during migration
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan
+            3. Start migration
+        expectedResults:
+            1. Mapping created and visible in UI
+            2.
+            3. logs should not show password during migration
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_retry_plan():
+    """
+    title: OSP: Test retry plan
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
```

Comment:
Small typo `from vmware to OSP`, please fix here and everywhere else it appears.

Reply:
I'll leave this as optional